commands/createfixpullrequests_test.go

```diff
@@ -147,20 +147,14 @@ func TestFixPackageVersion(t *testing.T) {
 			t.Run(test.technology.ToString(), func(t *testing.T) {
 				cfg := test.fixPackageVersionCmd(test)
 				// Fix impacted package for each technology
-				fixVersionInfo := packagehandlers.NewFixVersionInfo(test.fixVersion, test.technology, true)
+				fixVersionInfo := utils.NewFixVersionInfo(test.fixVersion, test.technology, true)
 				assert.NoError(t, cfg.updatePackageToFixedVersion(test.impactedPackaged, fixVersionInfo))
 				file, err := os.ReadFile(test.packageDescriptor)
 				assert.NoError(t, err)
 				assert.Contains(t, string(file), test.fixVersion)
 				// Verify that case-sensitive packages in python are lowered
 				assert.Contains(t, string(file), strings.ToLower(test.impactedPackaged))
 			})
-			t.Run(test.technology.ToString(), func(t *testing.T) {
```

Comment:
This was removed since I added that after inserting the new field.
Now we need to handle each indirect dependency separately when we will implement them, and add specific tests for them.